app/models/ontology/validations.rb

```diff
@@ -0,0 +1,16 @@
+module Ontology::Validations
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.